Exercicio06.js

```diff
@@ -0,0 +1,18 @@
+let firstGrade = prompt("digite a primeira nota:")
+let secondGrade = prompt("digite a segunda nota:")
+let thirdGrade = prompt("digite a terceira nota:")
+let fourthGrade = prompt("digite a quarta nota:")
+let fifthGrade = prompt("digite a quinta nota:")
+resultado = (Number(firstGrade) + Number(secondGrade) + Number(thirdGrade) + Number(fourthGrade) + Number(fifthGrade) ) / 5
+if(resultado >7){
+    console.log("aprovado(a)")
+}
+else if(resultado <5){
+    console.log("reprovado(a)")
+}
+else if(resultado =5 <=7 ){
```

Comment:
Não se esqueça que pra fazer comparações vc precisa usar `===` ou `==`. Se for o caso de usar junto o operador de maior ou menor usa-se apenas =, mas precisa encadear com o operador lógico
Neste caso ficaria assim:
```
else if(resultado >=5 && resultado <7 ){
```